Examples/Tests/ElectrostaticDirichletBC/analysis.py

```diff
@@ -30,7 +30,7 @@
 for ii, file in enumerate(files):
     ds = yt.load( file )
     times[ii] = (
-        ds.current_time.item() - float(ds.parameters.get('warpx.const_dt'))
+        ds.current_time.item()
```

Comment:
Is this change related to this PR?

Reply:
yes. Because we changed when we update cur_time